pkg/nexus/nexus_test.go

```diff
@@ -46,14 +46,56 @@ func TestDownloadFromNexusServer(t *testing.T) {
 	}
 }
 
+func TestDownloadFromNexus3Server(t *testing.T) {
+	b, err := createZipFile()
+	if err != nil {
+		t.Error(err.Error())
+	}
+	zipFileName := "leveransepakke.zip"
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Length", fmt.Sprintf("%d", len(b.Bytes())))
+		w.Header().Set("Content-Type", "application/zip")
+		w.Header().Set("Server", "Nexus/3.18.1-01 (PRO)")
+		w.Write(b.Bytes())
```

Comment:
Her burde du sjekke om du lager fornuftige spørringer mot nexus. Gjerne sjekk om snapshot vs releases fungerer som det skal

Reply:
Tester tilsvarer noenlunde eksisterende tester for nexus 2.  Kompleksiteten i APIene ligger hovedsakelig i samspillet med "ekte" Nexus. Det vil føre for langt å gjenskape ekte Nexus i mocker, men skal se om jeg kan verifisere noe mer i testene.